src/components/Container/index.tsx

```diff
@@ -0,0 +1,11 @@
+import "./index.css";
```

Comment:
You will need to rename index.css to index.module.css

```suggestion
import styles from "./index.module.css";
```